.changeset/dirty-beans-explain.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/core-app-api': minor
+'@backstage/frontend-app-api': minor
+'@backstage/plugin-app': minor
+---
+
+Add an app.signOutUrl option for configuring redirect upon sign out
```

Comment:
Thinking about my `auth` vs `app` comment some more I think where I'm coming from is discoverability, I would not look under `app` for this BUT if we added some documentation about this feature and how it works that would solve it for me to be honest. Can we do that, please?

We could add it here: https://backstage.io/docs/auth/identity-resolver or create a section of it's own under "Auth and Identity".

Reply:
💯 on discoverability. I failed to mention it in this PR (my bad), but we definitely need documentation for this. My intention was to float the technical direction here, and then follow up with doc updates through this same PR once the technical direction has been established.

Until now, configuring things like proxied auth requires external sources of information and/or experimentation to get it working.